### PR TITLE
Fix/#1516

### PR DESCRIPTION
**Pull request by hexyls**
closes https://github.com/protofire/omen-exchange/issues/1516

### CODE REVIEW

**Review by KadenZipfel — APPROVED** [review]

**Comment by Mi-Lan** [comment]
Was unable to test it since we can't create market on xDai and no markets had option to bond....

**Comment by hexyls** [comment]
@Mi-Lan a good way to test this is to checkout the branch, pop into [open.tsx](https://github.com/protofire/omen-exchange/blob/master/app/src/components/market/sections/market_profile/market_status/open.tsx#L253), and hardcode `currentTab` to `MarketDetailsTab.finalize`.

**Review by pimato — CHANGES_REQUESTED** [review]
@hexyls I´ve tried to bond some xdai on an outcome and it does not seem to work:

http://localhost:3000/#/0x1ec1bf127ceb4dd6c1d8640a65668d13b9eb09d9

Bildschirmfoto 2021-01-20 um 11 09 20

**Review by Mi-Lan — CHANGES_REQUESTED** [review]
@hexyls i'm getting same error as geronimo...Also should the amount on xDai be changed? Bonding 0.1 xDai is really nothing in my opinion? 0.1ETH in xDai makes more sense...
Screenshot 2021-01-20 at 13 19 58

**Comment by hexyls** [comment]
@Mi-Lan @pimato I think this error is because we are currently indexing markets that are using the old Realitio contracts. Should be resolved when the subgraph is updated 👍 In general this PR is cosmetic only so it should not impact functionality.

@Mi-Lan that is a great point about the bond amount! Updated to 10 xDAI 🙌

**Review by Mi-Lan — APPROVED** [review]

**Review by KadenZipfel — CHANGES_REQUESTED** [review]

**Review by KadenZipfel — APPROVED** [review]
LGTM!

File: app/src/components/market/sections/market_bond/market_bond.tsx [hunk 1]
```diff
@@ -61,8 +64,10 @@ const MarketBondWrapper: React.FC<Props> = (props: Props) => {
   const [isModalTransactionResultOpen, setIsModalTransactionResultOpen] = useState(false)
   const [outcomeIndex, setOutcomeIndex] = useState<number>(0)
   const probabilities = balances.map(balance => balance.probability)
+  const initialBondAmount =
+    networkId === networkIds.XDAI ? parseUnits('10', nativeAsset.decimals) : parseUnits('0.01', nativeAsset.decimals)
   const [bondEthAmount, setBondEthAmount] = useState<BigNumber>(
```
Comment (pRoy24) [inline app/src/components/market/sections/market_bond/market_bond.tsx:69]: Just a nomenclature change, we should probably change the state and setter to [bondNativeAssetAmount, setBondNativeAssetAmount]

File: app/src/components/market/sections/market_bond/market_bond.tsx [hunk 2]
```diff
@@ -153,7 +158,10 @@ const MarketBondWrapper: React.FC<Props> = (props: Props) => {
         <div>
           <>
             <CurrenciesWrapper>
-              <EthBalance value={`${formatNumber(formatBigNumber(ethBalance, TokenEthereum.decimals, 3), 3)}`} />
+              <EthBalance
```
Comment (pRoy24) [inline app/src/components/market/sections/market_bond/market_bond.tsx:161]: Similar this component should be renamed to <NativeAssetBalance ... />

**Review by pRoy24 — COMMENTED** [review]
Looks good, suggested a few nomenclature changes since the asset state variables and components to Native... instead of Eth... since these are no longer specific to ETH but general to the native asset on the chain.

**Review by pimato — APPROVED** [review]

**Pull request merged**